### DISCUSSION TITLE
Correlated features TreeSHAP vs KernelSHAP

### DISCUSSION THREAD

**Discussion by alielabridi** [shap/shap#1911] [Q&A]
I am currently working with some highly correlated features (text), and interested in exploring some path in how to get accurate shapley value estimate for those features. I am working a neural network model, and I currently use KernelSHAP (ExactExplainer) for it. I saw in this paper the following comment:

[EXPLAINING INDIVIDUAL PREDICTIONS WHEN FEATURES ARE
DEPENDENT:
MORE ACCURATE APPROXIMATIONS TO SHAPLEY VALUES](https://arxiv.org/pdf/1903.10464.pdf)

"For the non-linear models, these methods clearly outperformed the Tree SHAP method,
which, to the best of our knowledge, is the only Shapley approach which tries to handle dependence between features in the prediction explanation setting."

Does anyone know how TreeSHAP method is supposed to handle dependence better than the original Kernel SHAP method? Does the same apply for other method like DeepExplainer?

**Comment by dankirk95** [comment]
An answer to this would be appreciated!
